script.js

```diff
@@ -1,12 +1,421 @@
+    );
+    summary.setAttribute(`class`, `summaryP`);
+    summary.textContent = show.summary.replace(/(<([^>]+)>)/gi, "");
+
+    //features div
+    let br3 = document.createElement(`br`);
+    let br4 = document.createElement(`br`);
+    let br5 = document.createElement(`br`);
+    containerDiv.appendChild(br3);
+    containerDiv.appendChild(br4);
+    containerDiv.appendChild(br5);
+    let features = document.createElement(`div`);
+    containerDiv.appendChild(features);
+    features.setAttribute(`class`, `features`);
+
+    //film status
+    let statusP = paragraph();
+    features.appendChild(statusP);
+    statusP.setAttribute(`class`, `bottomDiv`);
+    statusP.textContent = `Status:  ${show.status}`;
+
+    //film language
+    let languageP = paragraph();
+    features.appendChild(languageP);
+    languageP.setAttribute(`class`, `bottomDiv`);
+    languageP.textContent = `Language:  ${show.language}`;
+  });
+}
+
+function codeCorrection(x) {
+  if (x < 10) {
+    return `0${x}`;
+  } else {
+    return x;
+  }
+}
+
+function search() {
+  let form = document.createElement(`form`);
+  form.setAttribute(`id`, `searchForm`);
+  header.appendChild(form);
+  let input = document.createElement(`input`);
+  form.appendChild(input);
+  input.setAttribute(`type`, `text`);
+  input.setAttribute(`id`, `searchInput`);
+  input.setAttribute(`class`, `searchInput`);
+  input.setAttribute(`placeholder`, `Search...`);
+  input.setAttribute(`title`, `Find Your Film Here`);
+  let searchBtn = document.createElement(`a`);
+  form.appendChild(searchBtn);
+  searchBtn.setAttribute(`class`, `search-btn`);
+  searchBtn.setAttribute(`href`, `#`);
+  searchBtn.setAttribute(`title`, `Find Your Film Here`);
+  searchBtn.textContent = `Search...`;
+
+  //search bar functionality
+  let searchBar = document.getElementById(`searchInput`);
+  searchBar.addEventListener(`keyup`, (x) => {
+    let searchValue = x.target.value.toLowerCase();
+    let searchItems = document.getElementsByClassName(`container`);
+
+    Array.from(searchItems).forEach((element) => {
+      let title = element.textContent;
+
+      let appearsOnText = title.toLowerCase().indexOf(searchValue) != -1;
+      console.log();
+      // console.log(element.className === `container episode`);
+      if (
+        appearsOnText &&
+        element.className === `container show` &&
+        (element.className === `container episode`) === false
+      ) {
+        element.style.display = "initial";
+      } else if (
+        appearsOnText &&
+        ((element.className === `container show`) === true).length +
+          ((element.className === `container show`) === false).length >
+          ((element.className === `container show`) === true).length
+        // &&  (element.className === `container episode`) === true
+      ) {
+        // if (
+        //   appearsOnText &&
+        //   element.className === `container show` &&
```

Comment:
before a pull request, remember to remove any code which is commented out and not used